kong/llm/drivers/azure.lua

```diff
@@ -114,7 +133,9 @@ function _M.configure_request(conf)
   local query_table = kong.request.get_query()
 
   -- technically min supported version
-  query_table["api-version"] = conf.model.options and conf.model.options.azure_api_version or "2023-05-15"
+  query_table["api-version"] = kong.request.get_query_arg("api-version")
+                            or (conf.model.options and conf.model.options.azure_api_version)
+                            or "2023-05-15"
```

Comment:
Can we leave a link or an explanation why this date marks the least supported version. This information may also be useful in the documentation and probably deserve some exposure.

Reply:
It's not needed, I think this is legacy from some revision ago (because azure_api_version is actually REQUIRED) so I removed it.